.buildkite/test-pipeline.yaml

```diff
@@ -48,6 +48,8 @@ steps:
   - VLLM_TEST_SAME_HOST=1 torchrun --nproc-per-node=4 distributed/test_same_node.py
   - TEST_DIST_MODEL=facebook/opt-125m DISTRIBUTED_EXECUTOR_BACKEND=ray pytest -v -s distributed/test_basic_distributed_correctness.py
   - TEST_DIST_MODEL=meta-llama/Llama-2-7b-hf DISTRIBUTED_EXECUTOR_BACKEND=ray pytest -v -s distributed/test_basic_distributed_correctness.py
+  - TEST_DIST_MODEL=facebook/opt-125m DISTRIBUTED_EXECUTOR_BACKEND=ray VLLM_USE_SPMD_WORKER=1 VLLM_USE_RAY_COMPILED_DAG=1 pytest -v -s distributed/test_basic_distributed_correctness.py
```

Comment:
QQ: we will pip install ray again here? In that case, maybe we should do it "after" all other tests below because we don't want to restrict the ray version yet until the integration becomes stable and remove `VLLM_USE_SPMD_WORKER`

Reply:
Looks like we don't need to pip install ray here. The tests are passing.